src/page/AlertingPage.tsx

```diff
@@ -93,13 +97,23 @@ const AlertingPageContent = () => {
           {alertError}
         </Alert>
       )}
+      {hasWriterRole && !canWriteAlerts && (
+        <InsufficientPermissions
+          message={
+            <>
+              You are not able to edit alerts because you are missing <code>alert.instances.external:write</code>{' '}
+              permissions
+            </>
+          }
+        />
+      )}
```

Comment:
Perhaps this could be consolidated with the alert above?

Reply:
The check above needs to check the metrics DS can be read in order to display the `AlertingPageContent` component, otherwise it will try to make the request to fetch alerts and fail. So we need to conditionally render according to this condition.

This message, on the other hand, should be rendered within `AlertPageContent` because we want to display the read-only list of alerts. This only indicates the inability to write them.

But in order to make it more consistent, I've consolidated the component used to display the message so that the text remains the same, only changing the list of missing permissions.